### PR TITLE
Safari column missing in Yari

### LINKED ISSUE #1676: Safari column missing in Yari
_Originally posted by @peterbe in https://github.com/mdn/yari/issues/1595#issuecomment-719761733_

Compare http://localhost:3000/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns with  https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns 
The "Safari" column is missing.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1676

What happened was that [`safari` was added to the list of `webextension-desktop`](https://github.com/mdn/kumascript/commit/56bb32e064b62252d6e7a069960724abef631366). 
The original PR was here: https://github.com/mdn/kumascript/pull/1396

I don't know exactly when we forked Compat.ejs into Yari, but clearly this is a case of a macro that was edit since *after* the fork point.

### CODE REVIEW

File: client/src/document/ingredients/browser-compatibility-table/headers.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const PLATFORM_BROWSERS: { [key: string]: bcd.BrowserNames[] } = {
     "samsunginternet_android",
   ],
   server: ["nodejs"],
-  "webextensions-desktop": ["chrome", "edge", "firefox", "opera"],
+  "webextensions-desktop": ["chrome", "edge", "firefox", "opera", "safari"],
```
Comment (peterbe) [inline client/src/document/ingredients/browser-compatibility-table/headers.tsx:16]: Visit http://localhost:3000/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns and enjoy the glorious presence of the "Safari" column.

File: client/src/document/ingredients/browser-compatibility-table/index.tsx [hunk 2]
```diff
@@ -34,7 +34,9 @@ const NEW_ISSUE_TEMPLATE = `
 </details>
 `;
 
-function gatherPlatformsAndBrowsers(category): [string[], bcd.BrowserNames[]] {
+function gatherPlatformsAndBrowsers(
+  category: string
```
Comment (peterbe) [inline client/src/document/ingredients/browser-compatibility-table/index.tsx:38]: Just a little drive-by edit I snook in when I noticed that parameter didn't have a type. Harmless.

File: kumascript/macros/Compat.ejs [hunk 3]
```diff
@@ -48,7 +48,7 @@ const browsers = {
   "desktop": ['chrome', 'edge', 'firefox', 'ie', 'opera', 'safari'],
   "mobile": ['webview_android', 'chrome_android', 'firefox_android', 'opera_android', 'safari_ios', 'samsunginternet_android'],
   "server": ['nodejs'],
-  "webextensions-desktop": ['chrome', 'edge', 'firefox', 'opera'],
+  "webextensions-desktop": ['chrome', 'edge', 'firefox', 'opera', 'safari'],
```
Comment (peterbe) [inline kumascript/macros/Compat.ejs:51]: Won't actually be used but now it's just a bit more complete.

**Review by escattone — APPROVED** [review]
Thanks @peterbe! I'm going to update our `kumascript` folder with the latest from https://github.com/mdn/kumascript later this week.
